tests/tests.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+require '../functions.php';
+
+class Tests extends TestCase
+{
+    public function testDisplayChairs_success1()
+    {
+        $test_array = [[
+            'chairname' => 'highchair',
+            'designer' => 'gary',
+            'designyear' => '2001'
+        ]];
+        $result = displayChairs($test_array);
+        $expected = '<h3>highchair' . '</h3>' . '<span>Designer: gary' . '</span><br/>' . '<span>Year: 2001' . '</span><br/>';
```

Comment:
Bear in mind that once you tweak the two div tags that aren't display you will need to update your expected result here to match!